### DISCUSSION TITLE
How to display token transfer instead of transfer function in the transaction summary?

### DISCUSSION THREAD

**Discussion by JOY** [blockscout/blockscout#9690] [Q&A]
Hello. In our Blockscout instance for DOS Chain, all the summary for token transfer wil display something like:
0x23...38FC called transfer on ContractName

Ex: https://doscan.io/tx/0xfc60dacdc463f71e7bee32d111d864ee1a63646870c0ce69a6965594dbaab2be

But in the Blockscout hosted one, it says:
Transfer 68.82 USDT to 0x0a...628a

Ex: https://optimism.blockscout.com/tx/0xc72620f15fda1a50f352657e9b33443b1dd707b784a40cd585c616fcceff2ba4?tab=index

How can I achive that summary which is much cleaner?

Thanks,
JOY

**Comment by GimluCom** [comment]
Looks like it has been solved?

**Reply by JOY** [reply]
Nope. It is the same as you can see in the first link.
![image](https://github.com/blockscout/blockscout/assets/5027251/83f28ed4-8b4a-4eee-a426-4e8c2aa8ce3e)
